### DISCUSSION TITLE
Question about installation procedure

### DISCUSSION THREAD

**Discussion by mauriciocirelli** [thanos-io/thanos#4207] [Questions & Answers]
Dear, 

I have installed prometheus and grafana on an Azure VM to monitor my servers.
It is working fine, but I need long-term storage and prometheus is already requiring too much RAM to run our queries and the monitoring system is getting slow. Doing my research I came up to this project.

Please, forgive me if this is not the right place to ask.

I would like to install thanos on this VM, without docker or kubernetes, point grafana to thanos to seamlessly migrate my current dashboards to thanos. And it would be desirable to store thanos database on a Azure Storage Account.

So, I have two questions:
1. How to install and setup thanos directly on a Ubuntu VM without docker or kubernetes? Every setup example I have found so far relies on docker or kubernetes.
2. Which kind of Azure Storage should I use? Blob Containers or File Shares?

Thank you!

**Comment by sevagh** [comment]
@mauriciocirelli I found that although a lot of the provided deployment material is aimed at container deployments, I run Thanos, Prometheus, and Alertmanager on bare metal servers using systemd with no problems. Go binaries can run almost anywhere.

1. Download the release tarball somewhere (amd64, probably?) - for our deployment, we store third-party binaries in an object store bucket and install them onto our production hosts using Ansible under `/opt/thanos/...`
2. Set up simple systemd file services e.g. /etc/systemd/system/thanos-query.service

It's really that simple. I can paste some example systemd files that I use for Prometheus and Thanos on Monday.

**Reply by mauriciocirelli** [reply]
Hi @sevagh,

Thank you for taking a look at my question.
I am pretty new to this. I have already setup an environment running Prometheus and Grafana (standalone) as systemd services.
I guess I do not need AlertManager, as I am using Grafana's alert system.

If you do not mind, I would like to see an example of your Thanos setup... It seems simple, indeed, but it would help me make sure I have not done anything wrong.

Also, regarding Azure storage: do you know if it works with an Azure Blob Container or an Azure File Share?
